File: test_api.js
async function run() {
  try {
    const loginRes = await fetch('http://localhost:5254/api/auth/login', {
      method: 'POST',
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify({ email: 'admin@gmail.com', password: '123' })
    });
    const { token } = await loginRes.json();

    const quoteRes = await fetch('http://localhost:5254/api/scrape-orders/quote', {
      method: 'POST',
      headers: { 
        'Content-Type': 'application/json',
        'Authorization': `Bearer ${token}`
      },
      body: JSON.stringify({ mentionsPackage: 'FULL_UNLIMITED' })
    });
    const data = await quoteRes.json();
    console.log('STATUS:', quoteRes.status);
    console.log('SUCCESS:', data);
  } catch (err) {
    console.log('ERROR:', err.message);
  }
}
run();

File: test_api.py
import urllib.request
import json

req1 = urllib.request.Request('http://localhost:5254/api/auth/login', data=json.dumps({"email":"admin@gmail.com","password":"123"}).encode(), headers={'Content-Type': 'application/json'})
resp1 = urllib.request.urlopen(req1)
token = json.loads(resp1.read())['token']

req2 = urllib.request.Request('http://localhost:5254/api/scrape-orders/quote', data=json.dumps({"mentionsPackage":"FULL_UNLIMITED"}).encode(), headers={'Content-Type': 'application/json', 'Authorization': f'Bearer {token}'})
try:
    resp2 = urllib.request.urlopen(req2)
    print("SUCCESS:")
    print(resp2.read().decode())
except Exception as e:
    print("ERROR:", e)
    if hasattr(e, 'read'):
        print(e.read().decode())
